test/_server.ts

```diff
@@ -24,6 +24,7 @@ const baseCreateServer = (fn): (() => Promise<TestServer>) => {
 		server.url = `http://${host}:${port}`;
 		server.protocol = 'http';
 		server.listen(port);
+		// @ts-ignore
```

Comment:
Don't use `@ts-ignore`. Resolve the type issue instead. Applies to everywhere where this is used.

You could see https://github.com/sindresorhus/pageres/pull/371/files for inspiration.